Nodejs/Product/Nodejs/Jade/Classifier/JadeClassificationTypes.cs

```diff
@@ -1,8 +1,8 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 namespace Microsoft.NodejsTools.Jade
 {
-    internal class JadeClassificationTypes
+    internal static class JadeClassificationTypes
```

Comment:
>JadeClassificationTypes [](start = 26, length = 23)

Consider making these private to JadeClassificationDefinitions.

Reply:
They are used in other places, so no real benefit to make that change